recipes/emio/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class EmioConan(ConanFile):
+    name = "emio"
+    description = "A character input/output library for embedded systems."
+    topics = ("format", "scan", "header-only")
+    homepage = "https://github.com/viatorus/emio"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
```

Comment:
```suggestion
    package_type = "header-library"
    no_copy_source = True
```

To be more ready for Conan 2.0

Reply:
Changed as suggested.